eZ/Publish/Core/SignalSlot/Signal/TrashService/DeleteTrashItemSignal.php

```diff
@@ -21,4 +21,18 @@ class DeleteTrashItemSignal extends Signal
      * @var mixed
      */
     public $trashItemId;
+
+    /**
+     * Identifier of content associated with deleted TrashItem.
+     *
+     * @var mixed
+     */
+    public $contentId;
+
+    /**
+     * Indicates that content was removed.
+     *
+     * @var bool
+     */
+    public $contentRemoved;
```

Comment:
Based on similar comment for `EmptyTrashSignal`, I'd go here just for `public \eZ\Publish\API\Repository\Values\Content\Trash\TrashItemDeleteResult $deleted`.

Please also mark `$trashItemId` as: `@deprecated Use <code>$deleted->trashItemId</code> instead`.
(Property naming debatable, I'm fine with `$trashItemDeleteResult` also or any other name that looks better to you).